packages/vm/src/testing/mock.rs

```diff
@@ -182,9 +189,12 @@ impl<C: DeserializeOwned> MockQuerier<C> {
 impl<C: DeserializeOwned> Querier for MockQuerier<C> {
     fn raw_query(&self, bin_request: &[u8]) -> QuerierResult {
         let res = self.querier.raw_query(bin_request);
-        let used_gas = (bin_request.len() + to_binary(&res).unwrap().len()) as u64;
+        // FIXME: is this correct? Was this charged already?
+        let gas_info = GasInfo::with_externally_used(
+            (bin_request.len() + to_binary(&res).unwrap().len()) as u64,
```

Comment:
Just realized this is MockQuerier.

Yes, this will be charged as external gas in the sdk. You could say `100_000 flat rate + 100 * res.len()` to be close to what the sdk charges (after GasMultiplier), but this is fine also. Maybe just a fixed rate - even 25025 or such - so it is more noticeable where it comes from in test cases.

Reply:
Thanks for the explanation. This is updated in https://github.com/CosmWasm/cosmwasm/pull/475/commits/0c7125731433f3ccaf05f719e245072c07cc2744 and can easily be adjusted to realistic values